nuxeo-pipeline-apb/templates/nuxeo-image-dockerfile.j2

```diff
@@ -1,7 +1,7 @@
 FROM {{ base_image_name }}:latest
 # Set up the /build folder with the install script, the marketplace package and the studio credentials
 USER root
-COPY marketplace.zip /build/marketplace
+RUN if [ -f marketplace.zip ] ; then cp marketplace.zip /build/marketplace ; fi
```

Comment:
Did you test that?

`RUN` happens in the container, not in the build environment. So that means there wille never be a `marketplace.zip` file... or there is something that I don't understand

Reply:
fixed with j2 templating